python/fledge/common/web/ssl_wrapper.py

```diff
@@ -80,21 +80,27 @@ def verify_against_revoked(cls):
     @classmethod
     def verify_against_ca(cls):
         echo_process = subprocess.Popen(['echo', cls.user_cert], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        a = subprocess.Popen(["openssl", "verify", "-CAfile", cls.ca_cert, "-x509_strict"], stdin=echo_process.stdout, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+        args = ["openssl", "verify", "-CAfile", cls.ca_cert]
+        cmd = "openssl verify -CAfile {}".format(cls.ca_cert)
+        # TODO: Add JIRA for to handle x509 strict check when open ssl version is 3 or so
+        # This is completely a workaround as of now CentOS9 stream has openssl version with 3.0
+        if utils.get_open_ssl_version(version_string=False)[0] < 3:
+            x509_strict = "-x509_strict"
+            args.append(x509_strict)
```

Comment:
can directly append instead of having extra var

Reply:
This variable is also used in `cmd` for logging purpose.